### ISSUE TITLE
Add check to ensure that a pod has a pod security policy specified

### ISSUE THREAD

**Issue by viswajithiii** [stackrox/kube-linter#37]
(Also validate that the PSP and security context are compatible. And in another check, perhaps validate that the PSP is as tight as possible.)

**Comment by garethahealy** [comment]
is this a dead policy, since PSP are now deprecated?
- https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/

**Comment by viswajithiii** [comment]
Yeah, pretty much. We are not planning to do it unless a lot of people explicitly ask for it (which doesn't seem to be happening).
